packages/manager/modules/pci/src/components/project/instance/instance.class.js

```diff
@@ -206,4 +206,27 @@ export default class Instance {
   isStopped() {
     return this.status === 'SHUTOFF';
   }
+
+  isShelved() {
+    return [
+      INSTANCE_STATUS.SHELVED,
+      INSTANCE_STATUS.SHELVED_OFFLOADED,
+    ].includes(this.status);
+  }
+
+  isShelving() {
+    return this.status === INSTANCE_STATUS.SHELVING;
+  }
+
+  isUnshelving() {
+    return this.status === INSTANCE_STATUS.UNSHELVING;
+  }
+
+  isValidToShelve() {
```

Comment:
```suggestion
  canBeShelved() {
```
or
```suggestion
  isShelvingAvailable() {
```

Reply:
my function name are not understandable ?

Reply:
It's understandable but not sure it is correct

Reply:
Thanks, changed.